### PR TITLE
New changes

### PR DESCRIPTION
**Pull request by OlaposiO**
This is the SQL Code

### CODE REVIEW

**Comment by OlaposiO** [comment]
> Are we for sure going with 3 columns of storing scores for each quiz length or did we wanna go with what ed mentioned yesterday of just scoring their total overall questions right regardless of what quiz length they took

Yeah so this would not be a score keeper per test. It would be a counter for how many times they took each test.

**Comment by vladi3ir** [comment]
This is a good working level with opportunity to refactor if needed. I'm OK with approving this.

File: UserScoreSQL.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+CREATE TABLE UserScoreData(
+Username VARCHAR(250),
+Score INT,
+QuizLength25 INT,
+QuizLength50 INT,
+QuizLength100 INT,
+CorrectAmount INT,
```
Comment (Eddonomochi) [inline UserScoreSQL.txt]: What is the diff between correct amount and score?

**Comment by Eddonomochi** [comment]
> > Are we for sure going with 3 columns of storing scores for each quiz length or did we wanna go with what ed mentioned yesterday of just scoring their total overall questions right regardless of what quiz length they took
> 
> Yeah so this would not be a score keeper per test. It would be a counter for how many times they took each test.

Cool, I like the changes you made to the table! Provides a clearer understanding and intent of the fields!

**Pull request merged**